design-reviews/recipes/engagement-process.md

```diff
@@ -0,0 +1,129 @@
+# Incorporating Design Reviews into an Engagement
+
+## Introduction
+
+Design reviews should not feel like a burden.  Design reviews can be easily incorporated into your dev crew process with minimal overhead.  
+
+- Only create design reviews when needed.  Not every story or task requires a full blown design review
+- Leverage this guidance and make changes to fit in best with your team.  Every team works a bit different and has different tweaks that work best for them.
+- Leverage SME's as needed during design reviews. Not every story needs SME or leadership sign-off.  Most design reviews can be fully executed within your own dev crew.
+
+The following is are some guidelines on how you and your team can start incorporating design reviews into your normal day to day agile process.
+
+## Table of Contents
+
+- [Envisioning / ADS](#envisioning-/-ads)
+- [Game Plan](#game-plan)
+- [Sprint Planning](#sprint-planning)
+- [Sprint Backlog Grooming](#sprint-backlog-grooming)
+- [Sprint Retrospectives](#sprint-retrospectives)
+- [Wrap-up Sprints](#wrap-up-sprints)
+
+## Envisioning / ADS
+
+Early on within an engagement process we are working with customers to understand their unique goals and objectives.  We are defining requirements to understand definition of done.  We are diving deep into their existing infrastructure and architecture to understand constraints.
+
+During this time we are uncovering as many unknowns as possible to help us jointly generate a design to meet the customers goals.  We are leveraging all the new found information an incorporating it into our game plan design for the customer engagement.
+
+> **Tip**: We have not uncovered all the unknowns at this point. You will continue to learn and discover more during the lifetime of your engagement
+
+## Game Plan
+
+The game plan document is our guiding star throughout the engagement.  The game plan document should include the following sections and answer the following questions.
+
+### Deliverable
+
+1. What are we delivering?
+1. What does success look like?
+1. What is the definition of done?
+1. What security, privacy and/or performance requirements are required?
+
+### Design
+
+1. What does the high level architecture look like?
+1. What does the technology stack look like?
+1. How does this solution fit into the customers overall product / service?
+
+### Process
+
+1. How will engineering fundamentals be leveraged during the project?
+1. What is the proposed sprint schedule?  How many sprints?
+1. Who is the team? CSE engineers? Customer Engineers?
+1. What is the sharing plan after the engagement is complete?
+
+### Ready to start
+
+1. Has the customer signed off on the plan?
+1. Are there any blocking issues to starting the engagement?
+1. What are the known risks?
+
+> **Tip**: Keep your game plan up to date as your design changes over time
+
+## Checkpoint Reviews

```

Comment:
Assuming no issues with describing this specific process in customer-facing playbook?

Reply:
@fnocera - thoughts?

Reply:
What does this have to do with design reviews? This seems excessive and out of scope.

Reply:
This is about handling design artifacts throughout the lifetime of an engagement.  The checkpoint review is a great opportunity to review relevant artifacts that came out of design reviews.